microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md

```diff
@@ -55,7 +84,8 @@ The following table lists the currently available permission strings to use in y
 | `cookies` | Gives your extension access to the `chrome.cookies` API. |
 | `debugger` | Gives your extension access to the `chrome.debugger` API. |
 | `declarativeContent` | Gives your extension access to the `chrome.declarativeContent` API. |
-| `declarativeNetRequest` | Gives your extension access to the `chrome.declarativeNetRequest` API. |
+| `declarativeNetRequest` | Gives your extension access to the `chrome.declarativeNetRequest` API. Allows the extension to block and upgrade requests without requiring host permissions, but requires host permissions to redirect requests or modify headers.|
+| `declarativeNetRequestWithHostAccess` | Gives your extension access to the `chrome.declarativeNetRequest` API.  Always requires host permissions on the request URL and on the initiator, to act on the request. |
```

Comment:
@utkarshspat - pls review the above two rows

Reply:
LGTM